src/lib.rs

```diff
@@ -59,13 +59,12 @@
 //!     connect(srv, cli);
 //! }
 //! ```
-
 #![cfg_attr(feature = "chan_select", feature(mpsc_select))]
 
 use std::marker;
 use std::thread::spawn;
 use std::mem::transmute;
-use std::sync::mpsc::{Sender, Receiver, channel};
+use std::sync::mpsc::{channel, Receiver, Sender};
```

Comment:
rustfmt also prefers imported functions listed before structs?

Reply:
It might just be that it orders them lexicographically in order to avoid ambiguity?

Reply:
But then Receiver should come before Sender, right?

Reply:
Good point :smile:. I wonder why it choses this ordering then....